components/ItemComponents/BreadcrumbsModule/index.js

```diff
@@ -105,9 +103,7 @@ const BreadcrumbsModule = ({
       <Breadcrumbs
         breadcrumbs={breadcrumbs.map(breadcrumb =>
           Object.assign({}, breadcrumb, {
-            title: breadcrumb.title
-              ? markdownit.renderInline(breadcrumb.title)
-              : UNTITLED_TEXT
+            title: breadcrumb.title ? breadcrumb.title : UNTITLED_TEXT
```

Comment:
Just sanity checking - you don't want rendered markdown in breadcrumbs?

Reply:
i'm passing the title as-is and markdownifying it [in the breadcrumbs module](https://github.com/dpla/dpla-frontend/blob/68139062d86fdf6ad1a334f0cd64cb9984b1ee6c/components/shared/Breadcrumbs/index.js#L16-L21)